### DISCUSSION TITLE
Zepeto IWP Button component 추가하는 방법 좀 알려주세요

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#24] [5. Q&A]
IWP 연동하기에서,
월드 상품 만들기에서 생성한 상품을 월드에서 판매하려면, 우선 Scene에 UI 버튼을 생성한 후,
생성된 버튼에 AddComponent → Zepeto IWP Button component를 추가하면 된다고 하는데,
팝업 매뉴에 보이질 않습니다. 찾는 방법이나 해당 스크립트의 코드를 좀 알려주세요.

**Comment by jooseungyou** [comment]
답변에 앞서 혹시 zepeto.iwp package는 잘 설치 되어 있으실까요? 만약 설치 되어 있으시다면 버전은 어떻게 될까요?

**Reply by runtostar** [reply]
예 설치는 잘 되었고, 아래분 답변 방식으로 했더니 이제 잘 되네요. 감사합니다.

**Comment by x4cf** [comment]
홈페이지에서 월드 등록을 하신다음 월드 상품을 먼저 만들어야 합니다.
프로젝트ID를 잘 기입하였을 경우 자동으로 연동되어 IWP 스크립트를 추가한다음 잠시 기다리면 홈페이지에서 제작한 아이템 ID 확인될겁니다.

**Reply by runtostar** [reply]
예 답변 감사드립니다. 말씀하신 방법으로 했더니 이제 잘 됩니다~!!!
